internal/performance/api_util_test.go

```diff
@@ -0,0 +1,42 @@
+package performance
```

Comment:
I'm not sure this needs to be tested. It's an the internal subpackage, where the code is all in service of benchmarking. The benchmarks should be sufficient, and if they don't work something fails. @dimroc might disagree with me on this though...?

Reply:
I agree with you. I've moved on to the latest commit, just didn't want to leave you hanging.